docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -17,6 +17,24 @@ This guide describes how to clone the Alluxio repository, compile the source cod
 - [Maven 3.3.9 or later](http://maven.apache.org/download.cgi)
 - [Git](https://git-scm.org/downloads)
 
+Alternatively, we have publishhed a docker image [alluxio/alluxio-maven](https://hub.docker.com/r/alluxio/alluxio-maven) with Java, Maven and Git installed to build Alluxio source code.
+For example, to build Alluxio with JDK8, checkout this image at tag `0.0.5-jdk8`:
+
+```console
+$ docker pull alluxio/alluxio-maven:0.0.5-jdk8
+```
+
+Create a container `alluxio-build` based on this image and get into this container to proceed:
+
+```console
+$ docker run -itd \
```

Comment:
what do you expect the user to do after following the instructions to build alluxio in a container? the git repo and compiled jars will not be available on the host machine once the project is compiled, so should this include a local mount to persist the git repo? i feel like the launched container wont be very useful besides running maven; the user won't be able to do much with the compiled repo in the container

another idea is to checkout the git repo locally, mount the repo, and then compile it in the docker container to have the same end state of compiling locally but without needing to install the specific versions of java and maven that we use. i've seen this workflow for other projects when users had trouble installing specific dependencies needed to compile the project

Reply:
applied your idea